backend/apid/apid.go

```diff
@@ -73,12 +73,12 @@ func New(c Config, opts ...Option) (*APId, error) {
 		Authenticator:       c.Authenticator,
 	}
 
-	var tlsConfig *tls.Config
+	var tlsClientConfig *tls.Config
 	var err error
 	if c.TLS != nil {
 		// TODO(palourde): We should avoid using the loopback interface
 		c.TLS.InsecureSkipVerify = true
-		tlsConfig, err = c.TLS.ToTLSConfig()
```

Comment:
NOTE: As of #2627 the `InsecureSkipVerify` option is no longer copied when fabricating a `TLS.Config`. See: [api/core/tls.go](https://github.com/sensu/sensu-go/blob/master/api/core/v2/tls.go#L31-L54).